docs/source/pcovr.rst

```diff
@@ -49,6 +49,7 @@ using the eigendecomposition of a modified covariance matrix
     .. automethod:: transform
     .. automethod:: predict
     .. automethod:: inverse_transform
+    .. automethod:: score
```

Comment:
random comment: could we use `:members:` instead of specifying every single method?

Reply:
I want to specify the two _fit strategies underneath fit(), so I've enumerated each individually.